src/tests/index.js

```diff
@@ -1,15 +1,16 @@
 import chai from 'chai';
 import chaiHttp from 'chai-http';
-import app from '../index.js';
+import app from '../index';
+
 chai.use(chaiHttp);
 chai.should();
 
 describe('App Tests', () => {
-   it('should return the homepage', (done) => {
-      chai.request(app).get('/')
-      .end((err,res) => {
-         res.should.have.status(200);
-         done();
-      })
-   });
+  it('should return the homepage', (done) => {
```

Comment:
### Nitpick

- Maybe more tests for what you worked on.

Reply:
My tests are now in `signup.test.js` but are commented due to they can't pass without signup route.
I have tested them locally and they are all passing.